src/prisma/mypy.py

```diff
@@ -155,11 +155,9 @@ def _handle_include(self, ctx: MethodContext) -> Type:
             if self.config.warn_parsing_errors:
                 # TODO: add more details
                 # e.g. "include" to "find_unique" of "UserActions"
-                if isinstance(exc, UnparsedExpression):
-                    err_ctx = exc.context  # pylint: disable=no-member
-                else:
-                    err_ctx = include_expr
-
+                err_ctx = (
+                    exc.context if isinstance(exc, UnparsedExpression) else include_expr
+                )  # pylint: disable=no-member
```

Comment:
See comment on src/prisma/generator/models.py.

It also localises our lint comment which I prefer.